index.test.js

```diff
@@ -745,6 +745,90 @@ describe('Deploy to ECS', () => {
         expect(mockEcsWaiter).toHaveBeenCalledTimes(0);
     });
 
+    test('registers the task definition contents and creates a CodeDeploy deployment with custom application, deployment group and description', async () => {
```

Comment:
It would be great if we can add test case for the default description as well.

Reply:
Yep, I'll add that.

Reply:
Looking again, the description parameter shouldn't be passed to the api at all if it's not set so the test `registers the task definition contents and creates a CodeDeploy deployment, waits for max 6 hours` covers this scenario. The description isn't passed to the action, and the description isn't sent to the API as checked in the assertion here https://github.com/aws-actions/amazon-ecs-deploy-task-definition/blob/master/index.test.js#L598.